kafka/channel/pkg/utils/util.go

```diff
@@ -29,6 +34,8 @@ import (
 
 const (
 	BrokerConfigMapKey           = "bootstrapServers"
+	TlsName                      = "tlsName"
+	TlsNamespace                 = "tlsNamespace"
```

Comment:
Oh, I think we should name these new `ConfigMap` keys:

* `AuthSecretName`
* `AuthSecretNamespace`

Reply:
Or `TlsSecretName`.

Reply:
`TlsSecretName` -> it supports both: TLS and SASL

Reply:
I was thinking `TlsSecretName` but `AuthSecretName` sounds better actually